infra/base-images/base-builder/precompile_centipede

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash -eu
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+echo -n "Precompiling centipede"
+
+# Add Clang 15.
+wget https://commondatastorage.googleapis.com/chromium-browser-clang/Linux_x64/clang-llvmorg-15-init-1995-g5bec1ea7-1.tgz --output-document=/clang-llvmorg-15-init-1995-g5bec1ea7-1.tgz
+mkdir /clang
+tar zxvf /clang-llvmorg-15-init-1995-g5bec1ea7-1.tgz -C /clang
+rm -f /clang-llvmorg-15-init-1995-g5bec1ea7-1.tgz
+
+apt-get install -y libssl-dev
+# Build Centipede with bazel
```

Comment:
End with period please.

Reply:
> First pass. I would actually prefer to leave out any documentation changes from this PR until centipede is ready for public use. 

That is a good point, I will make them a seperate PR.

Reply:
Fixed, I really need to write myself a period linter/formatter.